tests-legacy/Unit/Core/Cart/Adding/Product/AddStandardProductTest.php

```diff
@@ -30,6 +30,9 @@
 use LegacyTests\Unit\Core\Cart\AbstractCartTest;
 use Product;
 
+/**
+ * behat equivalent : Scenarii/Cart/Adding/Product/add_standard_product.feature
```

Comment:
https://english.stackexchange.com/questions/11775/what-is-the-plural-of-scenario